### PR TITLE
feat(client): add background animation for description of first challenge

### PR DESCRIPTION
**Pull request by ahmaxed**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

This feature would flash the background color of the description area on the first challenge after the project preview has been closed. 

related: #43668

### CODE REVIEW

**Review by ShaunSHamilton — DISMISSED** [review]
This LGTM 👍 

Nice one, Ahmad!

If I nit-picked, I would ask why just comment out the `#description` rule? I would remove it.

File: client/src/templates/Challenges/classic/editor.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#description {
+/* #description {
```
Comment (ahmadabdolsaheb) [inline client/src/templates/Challenges/classic/editor.css:1]:
```suggestion
```

File: client/src/templates/Challenges/classic/editor.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
   background: var(--secondary-background);
-}
+} */
```
Comment (ahmadabdolsaheb) [inline client/src/templates/Challenges/classic/editor.css:3]:
```suggestion
```

File: client/src/templates/Challenges/classic/editor.css [hunk 3]
```diff
@@ -1,6 +1,4 @@
-#description {
   background: var(--secondary-background);
```
Comment (ahmadabdolsaheb) [inline client/src/templates/Challenges/classic/editor.css:1]:
```suggestion
```

**Review by ShaunSHamilton — APPROVED** [review]
👍

**Review by naomi-lgbt — APPROVED** [review]
I like it. This looks good.

**Pull request merged**